src/Sylius/Bundle/AdminBundle/Resources/views/ShippingMethod/Field/calculator.html.twig

```diff
@@ -0,0 +1 @@
+{{ data|humanize }}
```

Comment:
Maybe it can be named ``humanize.html.twig``, placed in some generic folder (``views/Grid/Field/``?) as it can be used in many places, not only in ShippingMethod?